src/java.base/share/classes/java/util/zip/ZipInputStream.java

```diff
@@ -191,6 +192,204 @@ public int available() throws IOException {
         }
     }
 
+    /**
+     * Reads a byte of uncompressed data from the input stream for the current
+     * ZIP entry. This method will block until enough input is available for
+     * decompression.
+     * @return the byte read, or -1 if end of compressed input is reached
+     * @throws    IOException if an I/O error has occurred
+     */
+    @Override
+    public int read() throws IOException {
+        return super.read();
+    }
+
+    /**
+     * Reads all remaining bytes from the input stream for the current ZIP entry.
+     * This method blocks until all remaining bytes have been read and end of
+     * stream is detected, or an exception is thrown. This method does not close
+     * the input stream.
+     *
+     * <p> When this stream reaches end of stream, further invocations of this
+     * method will return an empty byte array.
+     *
+     * <p> Note that this method is intended for simple cases where it is
+     * convenient to read all bytes into a byte array. It is not intended for
+     * reading input streams with large amounts of data.
+     *
+     * <p> The behavior for the case where the input stream is <i>asynchronously
+     * closed</i>, or the thread interrupted during the read, is highly input
+     * stream specific, and therefore not specified.
+     *
+     * <p> If an I/O error occurs reading from the input stream, then it may do
+     * so after some, but not all, bytes have been read. Consequently, the input
+     * stream may not be at end of stream and may be in an inconsistent state.
+     * It is strongly recommended that the stream be promptly closed if an I/O
+     * error occurs.
+     *
+     * @throws OutOfMemoryError {@inheritDoc}
+     *
+     * @since 9
+     */
+    @Override
+    public byte[] readAllBytes() throws IOException {
+        return super.readAllBytes();
+    }
+
+    /**
+     * Reads up to a specified number of bytes from the input stream
+     * for the current ZIP entry. This
+     * method blocks until the requested number of bytes has been read, end
```

Comment:
Nit - some of these words could be moved to the previous line to make the line length consistent with other lines.

Reply:
I had addressed it but had not pushed it, thanks for the reminder